recipes/lucit-licensing-python/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "lucit-licensing-python" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LUCIT-Systems-and-Development/lucit-licensing-python/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 49eb84c50772d25e8723ac8d39c7cbae29ab90215d173d7b84fd1c2440fae9c7
+
+build:
+  skip: true  # [py<37]
```

Comment:
```suggestion
```
This can be removed as conda-forge does not build for python3.6

Reply:
Done, thanks!